CDS_Dashboard/script.js

```diff
@@ -0,0 +1,552 @@
+var chart;
+var chart2;
+$('#exampleModal').modal('hide');
+
+
+$(document).ready( function() {
+ var mq = window.matchMedia("(max-width: 600px)");
+ var desktopMq = window.matchMedia("(max-width: 800px)");
+ var desktopMq2 = window.matchMedia('(max-width: 1290px)');
+ mq.addListener(widthChange);
+ desktopMq.addListener(widthChange);
+ desktopMq2.addListener(widthChange);
+ 
+ widthChange(mq, desktopMq, desktopMq2);
+
+function widthChange(mq, desktopMq, desktopMq2) {
+    if (mq.matches) {
+        console.log('should be mobile');
+        $('.container').css('width', '550px')
+        $('#secondContainer').css("height", "250px");
+        $('#secondContainer').css('width', '300px');
+        $('#secondContainer').css('display', 'block');
+        $('#chartContainer').css("height", "250px");
+        $('#chartContainer').css("width", '300px');
+    } else if (desktopMq.matches) {
+        console.log('should be desktop');
+        $('#secondContainer').css("height", "500px");
+        $('#secondContainer').css('width', '300px');
+        $('#chartContainer').css("height", "500px");
+        $('#chartContainer').css('width', '350px');
+    } else if (desktopMq2.matches) {
+        $('#secondContainer').css("height", "500px");
+        $('#secondContainer').css('width', '350px');
+        $('#chartContainer').css("height", "500px");
+        $('#chartContainer').css('width', '400px'); 
+    } 
+    else {
+        $('#secondContainer').css('width', '500px');
+        $('#chartContainer').css('width', '500px');
+    }
+}
+
+var title = 'Projects';
```

Comment:
This line, and the lines after it (except the last) need either to start with `var` or end in a comma instead of semi-colon. You're getting the proper scope here on `title